data-esnext.js

```diff
@@ -3483,6 +3483,261 @@ exports.tests = [
       ie11: false,
   }
 },
+{
+  name: 'Object.fromEntries',
+  significance: 'small',
+  spec: 'https://github.com/tc39/proposal-object-from-entries',
+  category: STAGE1,
+  exec: function () {/*
+    var object = Object.fromEntries(new Map([['foo', 42], ['bar', 23]]));
+    return object.foo === 42 && object.bar === 23;
+  */},
+  res: {
+  }
+},
+{
+  name: 'getting last item from array',
+  spec: 'https://github.com/keithamus/proposal-array-last',
+  category: STAGE1,
+  significance: 'small',
+  subtests: [
+    {
+      name: 'Array.prototype.lastItem',
+      exec: function () {/*
+        return [1, 2, 3].lastItem === 3;
+      */},
+      res: {
+      }
+    },
+    {
+      name: 'Array.prototype.lastIndex',
```

Comment:
this is highly likely to go away, fwiw

Reply:
I don't see anything about it in the repo of the proposal.

Reply:
But I agree, I don't see any use cases for this getter.

Reply:
you're right, there's nothing in there yet. but `[].lastIndex` being `0` is super weird, and some on the committee were uncomfortable with it.